core/store/src/metrics.rs

```diff
@@ -0,0 +1,12 @@
+use near_metrics::{try_create_histogram_vec, HistogramVec};
+use once_cell::sync::Lazy;
+
+pub(crate) static DATABASE_GET_LATENCY_HIST: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_database_get_latency_us",
+        "Database read latency in microseconds, as a histogram",
+        &["column"],
+        Some(vec![20., 100., 200., 500., 800., 1000., 2000., 4000., 8000., 100000.]),
+    )
+    .unwrap()
+});
```

Comment:
As aforementioned, use base units in metrics (seconds rather than
microseconds).

This might work better with two labels, `&["op", "column"]`:

```rust
pub(crate) static DATABASE_OP_LATENCY_HIST: Lazy<HistogramVec> = Lazy::new(|| {
    try_create_histogram_vec(
        "near_database_op_latency_by_op_and_column",
        "Latency of database operations by operation and column.",
        &["op", "column"],
        Some(vec![0.02, 0.1, 0.2, 0.5, 0.8, 1.0, 2.0, 4.0, 8.0, 10.]),
    )
    .unwrap()
});
```

We can start with `get` operation but this can be naturally expanded
to include `set` in the future.

Also, why jump from 8 to 100 seconds at the end?

Reply:
8 and 100 are milliseconds not seconds.
I expect we never hit 100ms for a single get so that bucket will help to ensure that.

Reply:
To be honest I don’t find the choice of buckets convincing.  It feels to me that we should just stick with prometheus::exponential_buckets unless we have some data to demonstrate that different buckets would be better.